src/github/logic.py

```diff
@@ -77,8 +78,44 @@ def _extract_mentions(text: str) -> List[str]:
     return re.findall(GITHUB_MENTION_REGEX, text)
 
 
+def _expand_team_mentions(mentions: List[str]) -> Set[str]:
+    """Expand team mentions to their member usernames, supporting @org/team-slug.
+
+    Only expands mentions that follow the @org/team-slug format.
+    Other mentions are treated as regular user mentions.
+    """
+    expanded_mentions = set()
+    for mention in mentions:
+        # Only expand mentions that follow the org/team-slug format
+        if "/" in mention:
+            org, team_slug = mention.split("/", 1)
+            try:
+                team_members = github_graphql_client.get_team_members(org, team_slug)
+                if team_members:
+                    expanded_mentions.update(team_members)
+                else:
+                    expanded_mentions.add(mention)
+            except Exception as e:
+                logger.warning(f"Failed to expand team mention @{mention}: {e}")
+                expanded_mentions.add(mention)
+        else:
+            # Treat as regular user mention
+            expanded_mentions.add(mention)
+    return expanded_mentions
+
+
 def _pull_request_body_mentions(pull_request: PullRequest) -> List[str]:
-    return _extract_mentions(pull_request.body())
+    """Extract and expand mentions from PR body.
+
+    Args:
+        pull_request: The pull request to extract mentions from
+
+    Returns:
+        List of GitHub usernames mentioned in the PR body
+    """
+    mentions = _extract_mentions(pull_request.body())
+    expanded_mentions = _expand_team_mentions(mentions)
```

Comment:
Hmm, maybe a minor nit for me is to treat member mentions differently than team mentions. Coupling the two regex expressions together makes it a little harder to understand vs. if you had a separate `_extract_team_mentions` function to union with the original mentions set.